tracer/test/Datadog.Trace.TestHelpers/SpanMetadataRulesHelpers.cs

```diff
@@ -22,24 +23,29 @@ public static partial class SpanMetadataRules
 #pragma warning disable SA1201 // Elements should appear in the correct order
```

Comment:
Perhaps this class deservers it's own file now.

Reply:
Good idea, done in c0dec272d09afdd8ac4538321fa410d7a7f42ec3